openy_af4_vue_app/src/components/steps/SelectPath.vue

```diff
@@ -72,13 +72,22 @@ export default {
     backgroundImage: {
       type: Object,
       required: true
+    },
+    bsVersion: {
+      type: Number,
+      required: true
     }
   },
   data() {
     return {
       image: this.backgroundImage.mobile
     }
   },
+  computed: {
+    pathClasses() {
+      return this.bsVersion === 4 ? 'col-6 col-md-3' : 'col-xs-6 col-md-3'
```

Comment:
It was and should be `col-xs-6 col-sm-3` for BS3.

Reply:
Done in new commit